src/testUtils.ts

```diff
@@ -370,3 +373,7 @@ export const isTickInitialized = async (
     })
   ).returns
 }
+
+export const liquidity = (value: bigint) => {
```

Comment:
The function name should be more clearer, if it applies the liquidity decimal to value maybe consider naming it `toLiquidity`.